coalib/results/Diff.py

```diff
@@ -176,7 +197,7 @@ def unified_diff(self):
         """
         return ''.join(difflib.unified_diff(
             self.original,
-            self.modified,
+            self.modified if not self.delete else [],
```

Comment:
`if not self.delete else []` isn't needed since `modified` already takes care of this, right?